assembly/src/parsers/u32_ops.rs

```diff
@@ -360,16 +360,23 @@ pub fn parse_u32and(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), Ass
     Ok(())
 }
 
-/// Translates u32checked_or assembly instruction to VM operation `U32OR`.
+/// Translates u32checked_or assembly instruction to a sequence of VM operations.
 ///
-/// Implemented as: `U32OR` (1 VM cycle).
+/// Implemented as: `DUP1 DUP1 u32XOR MOVDN2 U32AND U32XOR` (6 VM cycles).
```

Comment:
> I don't think this "Implemented as" comment is very useful, since we can just read it below.
Agreed. I've also changed it for some other instructions in the same file.